lib/registration.pm

```diff
@@ -565,25 +567,19 @@ sub process_scc_register_addons {
     }
 }
 
-sub process_scc_register {
```

Comment:
Moved at the end of process_modules

Reply:
I find this more clear like this, "process_scc_register" is confusing : it handles the modules, so it makes sense to put it in process_modules. otherwise renaming it would make two different functions that handle modules, so would be equally confusing.